hack/update-generated-swagger-docs.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
We need to first run update docs and then run update spec. Where should I put that dependency? I already edit the order on pre-commit. Isn't it enough?